### PR TITLE
substrate: bump to rc6

### PR DESCRIPTION
**Pull request by Centril**

### CODE REVIEW

**Review by aafshar-poly — APPROVED** [review]

**Review by ParnianAlimi — APPROVED** [review]
The CI is happy, so I approve!

**Comment by Centril** [comment]
@maxsam4 What's the process for merging in things here?

**Comment by aafshar-poly** [comment]
@Centril once myself or @ParnianAlimi approve it, it is good to merge from the research team's side. If you want extra approval from your team, you can definitely wait for their input.

**Pull request merged**
